app/src/main/java/ch/epfl/culturequest/backend/map_collection/OTMLatLng.java

```diff
@@ -36,6 +38,26 @@ public double longitude() {
         return lon;
     }
 
+
+    //the following getters and setters are necessary unfortunately for us to be able to store this object
+    //in the database. this is because firebase reads the function name to store stuff. Refactoring the above functions makes many modifs to the porject
+    public double getLat() {
+        return latitude();
+    }
+
+    public double getLon() {
+        return longitude();
+    }
```

Comment:
Would it really be too cumbersome to change? In android Studio you can press Shift+F6 on the name of a function to refactor it all across a codebase. I'm not sure to see why you wouldn't do that